src/test/java/software/amazon/awssdk/crt/test/Mqtt5ClientTest.java

```diff
@@ -1362,69 +1362,6 @@ public void ConnNegativeID_UC7() {
         }
     }
 
-    /* Double Client ID disconnect and then reconnect test */
```

Comment:
Removing the double client ID reconnect test. It wasn't part of the testing plan anyway and it is **extremely** flakey and inconsistent. We know reconnect works and I don't think it is adding anything but it does cause CI to fail (not only here, but in other PRs too).

We can always add it back if there's a need but for now, I'm just going to remove it.